plugins/rikki/heroeslounge/models/Match.php

```diff
@@ -9,6 +9,7 @@
 use Rikki\Heroeslounge\Models\Season as Season;
 use Rikki\Heroeslounge\Models\Playoff;
 use Rikki\Heroeslounge\Models\Team;
+use Rikki\Heroeslounge\Models\Division as Division;
```

Comment:
Is this `as` not redundant since that's what it is already? (Same for season above)

Reply:
It is